### PR TITLE
Separate EasyTabInfo into two structs.

### PR DESCRIPTION
**Pull request by serge-rgb**
1. Leave only Position and Pressure info in EasyTab. Which allows for it to be moved to the top where the public interface is declared.
2. Created EasyTabContext to hold the guts and live inside EASYTAB_IMPLEMENTATION.

I don't know if commit comments will be shown in the pull request. So:
- Removed MaxPressure from the public-facing struct. Pressure ranges in [0,1], so MaxPressure is currently only useful inside EasyTab_Handle() when doing the divide.
- Renamed struct member `Context` to `WintabContext` because `EasyTabContext->Context` was confusing

(Edit: Yup. Comments do show up in the Pull Request. Today I Learned :)

### CODE REVIEW

**Comment by ApoorvaJ** [comment]
All things considered, are you sure there's value to splitting the struct? Hurts simplicity, right?

If we do decide to split the struct, I'd like RangeX, RangeY and MaxPressure in the EasyTab struct. While we aren't really using them right now, I think it makes sense to have them in the user-facing struct because they contain relevant data, and because they are platform-agnostic.

Let me know what you think about splitting vs keeping the simplicity and moving it back to the header section.

**Comment by serge-rgb** [comment]
Yup, it hurts simplicity. It turns one global into two globals and that's definitely not good.

So, here is what is bothering me. If it bothers you too maybe we can come up with a better solution.

When I was diving into the code the first time, I looked at the sample code and the function definitions. It took my a sec to realize that I could do EasyTab->PosX, etc to grab tablet info. It changed my mind about the cost of having a global, because it's pretty neat to access tablet info that way. It's clean, simple and flexible.

The reason it took me a minute to "get it" is that I didn't pay enough attention to the documentation the first time through. I went directly to the code examples (as I imagine many people do). The usage of the EasyTab var didn't register in my brain because there is no mention of it in the examples. It's right there in line 27. But I went straight to the examples. :) 

As it stands now, if I want to know the information that I can access from EasyTab, I need to scroll down to line 524, where I will see PosX, PosY, Pressure, Range(X|Y), etc. 

That's the main thing that I would like to fix. I would like the usage to be crystal clear, with the public interface defined right away. I don't want to skip 500 lines before I find what I came in for. IMO the cost of the split is worth it just to get the public interface declared right at the top of the file. Self-documenting code.

Anywho...

I can see why you don't like the split, most programmers know the search command in their editor and can skip the boilerplate to get to the contents of EasyTab.

Let me propose something simpler: Add a clear description of the relevant members of EasyTab in the code examples. I was a first-time user a couple days ago, and integration would have been much faster with that simple change. It was really the only thing that caused any friction for me.

In other words, change this (line 63):

```
            if (EasyTab_HandleEvent(Window, Message, LParam, WParam) == EASYTAB_OK) // Event
            {
                return true; // Tablet event handled
            }
```

into this

```
            if (EasyTab_HandleEvent(Window, Message, LParam, WParam) == EASYTAB_OK) // Event
            {
                // Use the EasyTab variable to access the tablet state:
                //    EasyTab->PosX 
                //    EasyTab->PosY
                //    EasyTab->Pressure
                // For more information, look for the EasyTabInfo struct.
            }
```

and do the same thing for the Linux example.

I think this is a much simpler change that will make us both happy. Let me know what you think.

**Comment by ApoorvaJ** [comment]
Done. I changed the location and content of the example comments a little bit, but I think it's pretty clear now. Thanks for the usability insight. :)
